host/sgx/switchless.c

```diff
@@ -222,22 +235,21 @@ oe_result_t oe_start_switchless_manager(
         {
             OE_RAISE(OE_THREAD_CREATE_ERROR);
         }
+
+        // Wait until the enclave worker thread has started.
+        // If so, spin_cout/total_spin_count will be non zero.
```

Comment:
I would change this to "spin_cout and total_spin_count". For I while I was confused because I thought it was saying "spin_count divided by total_spin_count"

Reply:
:) Fixed.